logs/lib/src/time.rs

```diff
@@ -0,0 +1,25 @@
+/// Gets the second unix timestamp for the stat filename
+#[must_use]
+pub fn millisecond_ts() -> u64 {
+    time::millisecond_ts()
+}
+
+#[cfg(target_os = "linux")]
+mod time {
+    use libc::{clock_gettime, timespec, CLOCK_REALTIME};
+    use std::mem;
+
+    /// Invokes `clock_gettime` from time.h in libc to get a `timespec` struct
+    fn get_time() -> timespec {
+        let mut tp: timespec = unsafe { mem::zeroed() };
+        unsafe {
+            clock_gettime(CLOCK_REALTIME, &mut tp);
+        }
```

Comment:
It's best practice to have SAFETY: reasoning comments for every unsafe block in rust that explains your reasoning for why something is safe

Reply:
it's an FFI call to a `libc` function, but I'll add a comment